cms/djangoapps/contentstore/tests/test_libraries.py

```diff
@@ -527,9 +527,7 @@ def test_creation(self):
         self._login_as_non_staff_user(logout_first=False)
         self.assertFalse(CourseCreatorRole().has_user(self.non_staff_user))
         with patch.dict('django.conf.settings.FEATURES', {'ENABLE_CREATOR_GROUP': True}):
-            lib_key2 = self._create_library(library="lib2", display_name="Test Library 2")
-            library2 = modulestore().get_library(lib_key2)
-            self.assertIsNotNone(library2)
+            lib_key2 = self._create_library(library="lib2", display_name="Test Library 2", expected_code=403)
```

Comment:
by getting rid of L531-532, you're also losing some functionality of the test, i.e. asserting that the library was actually added to the modulestore. Why do you need to change this file at all? It looks like your changed test doesn't call this method at all